src/services/server/ServerService.ts

```diff
@@ -39,6 +43,26 @@ export class ServerService {
     });
   }
 
+  getServer(server: IServerAddress): ng.IPromise<IServer> {
+    let deferred: ng.IDeferred<IServer> = this.$q.defer<IServer>();
+    this.dmrService.readAttribute({
+      address: this.generateAddress(server),
+      name: "server-state"
+    }).then(serverState =>{
+      if (serverState === "STOPPED") {
+        deferred.resolve(new Server(server, serverState, ''));
```

Comment:
@vblagoje Ok, that makes sense. However the only operations that can be executed on a stopped server, AFAIK, are get launch type and get status, but I guess this could change in the future.